src/test/java/uk/gov/hmcts/sptribs/caseworker/event/page/HearingRecordingUploadPageTest.java

```diff
@@ -27,80 +28,95 @@ public class HearingRecordingUploadPageTest {
+        AboutToStartOrSubmitResponse<CaseData, State> response = hearingRecordingUploadPage.midEvent(caseDetails, caseDetails);
 
-        //Then
         assertThat(response.getErrors()).hasSize(1);
+        assertThat(response.getErrors().get(0).contains("Please attach a mp3 document"));
     }
-
+    
     @Test
-    void shouldValidateUploadedDocumentWithoutCategoryWithoutDesc() {
-        //Given
-        final CaseDetails<CaseData, State> caseDetails = new CaseDetails<>();
-        final Listing listing = getRecordListing();
-        List<ListValue<CaseworkerCICDocument>> documentList = new ArrayList<>();
-        final CaseworkerCICDocument document = CaseworkerCICDocument.builder()
+    void midEventReturnsErrorForMissingDescription() {
+        this.documentList = new ArrayList<>();
+        CaseworkerCICDocument document = CaseworkerCICDocument.builder()
             .documentLink(Document.builder().filename("file.mp3").build())
+            .documentCategory(DocumentType.LINKED_DOCS)
             .build();
         ListValue<CaseworkerCICDocument> documentListValue = new ListValue<>();
         documentListValue.setValue(document);
-        documentList.add(documentListValue);
-        HearingSummary hearingSummary = HearingSummary.builder().recFile(documentList).build();
-        listing.setSummary(hearingSummary);
-
-        final CaseData caseData = CaseData.builder()
-            .listing(listing)
+        
+        this.documentList.add(documentListValue);
+        HearingSummary hearingSummary = HearingSummary.builder().recFile(this.documentList).build();
+        this.listing.setSummary(hearingSummary);
+        
+        CaseData caseData = CaseData.builder()
+            .listing(this.listing)
             .build();
-        caseDetails.setData(caseData);
-
-        //When
-        final AboutToStartOrSubmitResponse<CaseData, State> response = hearingRecordingUploadPage.midEvent(caseDetails, caseDetails);
-
-        //Then
-        assertThat(response.getErrors()).hasSize(2);
+        this.caseDetails.setData(caseData);
+        
+        AboutToStartOrSubmitResponse<CaseData, State> response = hearingRecordingUploadPage.midEvent(this.caseDetails, this.caseDetails);
+        
+        assertThat(response.getErrors()).hasSize(1);
+        assertThat(response.getErrors().get(0).contains("Description is mandatory for each document"));
     }
 
     @Test
-    void shouldValidateUploadedDocumentWithoutDoc() {
-        //Given
-        final CaseDetails<CaseData, State> caseDetails = new CaseDetails<>();
-        final Listing listing = getRecordListing();
-        List<ListValue<CaseworkerCICDocument>> documentList = new ArrayList<>();
-        final CaseworkerCICDocument document = CaseworkerCICDocument.builder()
+    void midEventReturnsErrorForMissingDocumentLink() {
+        this.documentList = new ArrayList<>();
```

Comment:
is this required? should be fine without

Reply:
It tests the if clause of `HearingRecordUploadPage.midEvent()`, line 55:
https://github.com/hmcts/sptribs-case-api/blob/1d9be6021eb4f88250f8179e54d6f1d609c92d99/src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/HearingRecordingUploadPage.java#L55